guide/14-deep-learning/how-named-entity-recognition-works.ipynb

```diff
@@ -16,24 +16,38 @@
     "<ul class=\"toc-item\">\n",
```

Comment:
<blockquote>A user can choose an appropriate backbone / library to train his/her model.</blockquote><p> </p><p>A user can choose an appropriate backbone to train the model.</p><p> </p><p>Refer to the section <a href="https://developers.arcgis.com/python/guide/install-and-set-up/#Install-deep-learning-dependencies" target="_blank">Install deep learning dependencies of arcgis.learn module</a><strong> </strong>for detailed <s>documentation</s> explanation about <s>on</s> <s>installation of the dependencies. </s>deep learning dependencies.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/799/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
fixed